examples/dgmulti_2d/elixir_navierstokes_convergence.jl

```diff
@@ -4,14 +4,14 @@ using Trixi
 ###############################################################################
 # semidiscretization of the ideal compressible Navier-Stokes equations
 
-reynolds_number() = 100
 prandtl_number() = 0.72
+viscosity() = 0.01
 
 equations = CompressibleEulerEquations2D(1.4)
 # Note: If you change the Navier-Stokes parameters here, also change them in the initial condition
 # I really do not like this structure but it should work for now
-equations_parabolic = CompressibleNavierStokesDiffusion2D(equations, Reynolds=reynolds_number(), Prandtl=prandtl_number(),
-                                                          Mach_freestream=0.5, gradient_variables=GradientVariablesPrimitive())
+equations_parabolic = CompressibleNavierStokesDiffusion2D(equations, viscosity=viscosity(), Prandtl=prandtl_number(),
```

Comment:
This same strategy is used in FLUXO (and my old codes with David). It is maybe weird, but at some point we need to set the Prandtl number because it is part of Fick's law.

Reply:
OK, then let's do it this way 🤷‍♂️